src/screens/MainScreen/index.js

```diff
@@ -18,16 +20,44 @@ const MainScreen = () => {
     getCalls();
   }, [currentUser]);
 
+  let testimo = null;
+  function smoothSlide(e) {
+    testimo.scrollIntoView({ behavior: "smooth" });
+  }
+
```

Comment:
Best to make it an arrow function for consistency

Reply:
done